pkg/synthetic/ticket.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package synthetic generates fake data for testing.
+package synthetic
+
+import (
+	"fmt"
+	"math"
+	"math/rand"
+
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	"open-match.dev/open-match/pkg/pb"
+	"open-match.dev/open-match/pkg/structs"
+)
+
+// Tickets generates artificial tickets with these properties:
+// mmr.rating = value 0 to 100, average 50, standard deviation of 15.
+// one of the role properties:
+//   role.dps = 1 with 40% chance
```

Comment:
Ideally these can be in some config that lives besides this file? so to tweak it, you can simply bring your own config? (Not blocking this PR on it - just a suggestion)

Reply:
This is already supported.  The tickets function merely is an easy to start with example.

Reply:
Added godoc line explaining as much.